e2e/my.test.js

```diff
@@ -0,0 +1,60 @@
+ const { log } = require('detox');
+
+describe('Example', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should have a visible button', async () => {
+    // log.error(device.appLaunchArgs.get())
+    await expect(element(by.traits(['button'])).atIndex(1)).toBeVisible();
+  });
+
+  it("New wallet button works", async () => {
+    let newWalletButton = element(by.text("CREATE NEW WALLET (NEW SEED)"));
+    await newWalletButton.tap();
+    await element(by.text("I HAVE SAVED \n THE SEED")).tap();
```

Comment:
When I run the test I got this error: (maybe it is something in my local repo)

C:\Users\juank\github\zingo-mobile-background-sync-android>yarn detox test -c android.emu.debug
yarn run v1.22.15
$ C:\Users\juank\github\zingo-mobile-background-sync-android\node_modules\.bin\detox test -c android.emu.debug
15:40:10.676 detox[88524] B jest --config e2e/jest.config.js
15:40:26.264 detox[87680] i my.test.js is assigned to emulator-5554 (Pixel_XL_API_33)
15:40:57.410 detox[87680] i Example: should have a visible button
15:41:00.961 detox[87680] i Example: should have a visible button [OK]
15:41:00.964 detox[87680] i Example: New wallet button works
15:41:05.023 detox[87680] i Example: New wallet button works [FAIL]

 FAIL  e2e/my.test.js (51.144 s)
  Example
    √ should have a visible button (3550 ms)
    × New wallet button works (4059 ms)

  ● Example › New wallet button works

    Test Failed: No views in hierarchy found matching: (an instance of android.widget.TextView and view.getText() with or without transformation to match: is "I HAVE SAVED \n THE SEED" and view has effective visibility <VISIBLE>)


    HINT: To print view hierarchy on failed actions/matches, use log-level verbose or higher.

      18 |     let newWalletButton = element(by.text("CREATE NEW WALLET (NEW SEED)"));
      19 |     await newWalletButton.tap();
    > 20 |     await element(by.text("I HAVE SAVED \n THE SEED")).tap();
         |                                                        ^
      21 |     // For some reason, the element that says 'ZEC', according to react,
      22 |     // has text of ' ---'
      23 |     // let zecText = element(by.text(" ---")).atIndex(1);

      at createErrorWithUserStack (node_modules/detox/src/utils/errorUtils.js:36:27)
      at Client._doSendAction (node_modules/detox/src/client/Client.js:164:32)
      at Client._sendMonitoredAction (node_modules/detox/src/client/Client.js:157:25)
      at Client.sendAction (node_modules/detox/src/client/Client.js:143:14)
      at Client.execute (node_modules/detox/src/client/Client.js:132:25)
      at InvocationManager.execute (node_modules/detox/src/invoke.js:12:39)
      at ActionInteraction.execute (node_modules/detox/src/android/interactions/native.js:23:31)
      at NativeElement.tap (node_modules/detox/src/android/core/NativeElement.js:37:97)
      at Object.tap (e2e/my.test.js:20:56)
      at asyncGeneratorStep (node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
      at _next (node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)

15:41:05.188 detox[88524] E Command failed with exit code = 1:
jest --config e2e/jest.config.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Reply:
Ooh, interesting.. I'll see if I can reproduce this!

If you get a chance, could you rerun this with more logging?
`yarn detox test -c android.emu.debug -l verbose`

Reply:
this is the result:
C:\Users\juank\github\zingo-mobile-background-sync-android>yarn detox test -c android.emu.debug -l verbose
yarn run v1.22.15
$ C:\Users\juank\github\zingo-mobile-background-sync-android\node_modules\.bin\detox test -c android.emu.debug -l verbose
14:04:47.045 detox[343380] i ws-server Detox server listening on localhost:56180...
14:04:47.238 detox[343380] B lifecycle jest --config e2e/jest.config.js
Determining test suites to run...
14:04:57.669 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\emulator\emulator.EXE" -list-avds --verbose
14:04:57.711 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\emulator\emulator.EXE" -version
14:04:58.071 detox[338528] i device Detected emulator binary version { major: 31, minor: 3, patch: 14, toString: [Function: toString] }
14:04:58.073 detox[338528] i device:ALLOCATE_DEVICE Trying to allocate a device based on "Pixel_XL_API_33"
14:04:58.077 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE"  devices
14:04:58.135 detox[338528] i child-process:EXEC_SUCCESS List of devices attached
R9JT10D7AFV     device
emulator-5554   device


14:04:58.138 detox[338528] i device:DEVICE_LOOKUP Device R9JT10D7AFV does not match "Pixel_XL_API_33"
14:04:58.151 detox[338528] i device:DEVICE_LOOKUP Found a matching & free device emulator-5554
14:04:58.155 detox[338528] i device:ALLOCATE_DEVICE Settled on emulator-5554
14:04:58.157 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "getprop dev.bootcomplete"
14:04:58.295 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "getprop ro.build.version.sdk"
14:04:58.371 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "settings put global animator_duration_scale 0"
14:04:58.507 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "settings put global window_animation_scale 0"
14:04:58.581 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "settings put global transition_animation_scale 0"
14:04:58.646 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "dumpsys power | grep \"^[ ]*m[UW].*=\""
14:04:59.327 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\build-tools\33.0.1\aapt.EXE" dump badging "C:\Users\juank\github\zingo-mobile-background-sync-android\android\app\build\outputs\apk\debug\app-debug.apk"
14:04:59.423 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "pm list packages org.ZingoLabs.Zingo"
14:04:59.492 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 uninstall org.ZingoLabs.Zingo
14:04:59.703 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "pm list packages org.ZingoLabs.Zingo.test"
14:04:59.786 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 uninstall org.ZingoLabs.Zingo.test
14:05:00.060 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "am force-stop org.ZingoLabs.Zingo"
14:05:00.263 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\build-tools\33.0.1\aapt.EXE" dump xmlstrings "C:\Users\juank\github\zingo-mobile-background-sync-android\android\app\build\outputs\apk\debug\app-debug.apk" AndroidManifest.xml
14:05:00.299 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\build-tools\33.0.1\aapt.EXE" dump xmlstrings "C:\Users\juank\github\zingo-mobile-background-sync-android\android\app\build\outputs\apk\androidTest\debug\app-debug-androidTest.apk" AndroidManifest.xml
14:05:00.349 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "rm -fr /data/local/tmp/detox"
14:05:00.448 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "mkdir -p /data/local/tmp/detox"
14:05:00.537 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 push "C:\Users\juank\github\zingo-mobile-background-sync-android\android\app\build\outputs\apk\debug\app-debug.apk" "/data/local/tmp/detox/Application.apk"
14:05:04.256 detox[338528] i child-process:SPAWN_CMD C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell pm install -r -g -t /data/14:05:10.408 detox[338528] i child-process:SPAWN_END C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell pm install -r -g -t /data/local/tmp/detox/Application.apk exited with code #0
14:05:10.409 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 push "C:\Users\juank\github\zingo-mobile-background-sync-android\android\app\build\outputs\apk\androidTest\debug\app-debug-androidTest.apk" "/data/local/tmp/detox/Test.apk"
14:05:10.537 detox[338528] i child-process:SPAWN_CMD C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell pm install -r -g -t /data/local/tmp/detox/Test.apk
14:05:10.895 detox[338528] i child-process:SPAWN_END C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell pm install -r -g -t /data/local/tmp/detox/Test.apk exited with code #0
14:05:10.897 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 reverse tcp:8081 tcp:8081
14:05:10.949 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "am force-stop org.ZingoLabs.Zingo"
14:05:13.018 detox[338528] i lifecycle my.test.js is assigned to emulator-5554 (Pixel_XL_API_33)
14:05:13.039 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "am force-stop org.ZingoLabs.Zingo"
14:05:13.120 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "date +\"%m-%d %T.000\""
14:05:13.204 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 reverse tcp:56180 tcp:56180
14:05:13.245 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "pm list instrumentation"
14:05:13.328 detox[338528] i child-process:SPAWN_CMD C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell am instrument -w -r -e detoxServer ws://localhost:56180 -e detoxSessionId b216e53d-f1a4-846e-d839-45f8c1333fd5 -e debug false org.ZingoLabs.Zingo.test/androidx.test.runner.AndroidJUnitRunner
14:05:13.830 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 shell "ps | grep \"org\.ZingoLabs\.Zingo$\""

14:05:20.936 detox[338528] i lifecycle Example: should have a visible button
14:05:24.714 detox[338528] i lifecycle Example: should have a visible button [OK]
14:05:24.716 detox[338528] i lifecycle Example: New wallet button works
14:05:29.095 detox[338528] i lifecycle Example: New wallet button works [FAIL]


14:05:29.145 detox[338528] i child-process:SPAWN_END C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE -s emulator-5554 shell am instrument -w -r -e detoxServer ws://localhost:56180 -e detoxSessionId b216e53d-f1a4-846e-d839-45f8c1333fd5 -e debug false org.ZingoLabs.Zingo.test/androidx.test.runner.AndroidJUnitRunner terminated with SIGINT
14:05:29.147 detox[338528] i child-process:EXEC_CMD "C:\Users\juank\AppData\Local\Android\Sdk\platform-tools\adb.EXE" -s emulator-5554 reverse --remove tcp:56180
 FAIL  e2e/my.test.js (33.508 s)
  Example
    √ should have a visible button (3778 ms)
    × New wallet button works (4378 ms)

  ● Example › New wallet button works

    Test Failed: No views in hierarchy found matching: (an instance of android.widget.TextView and view.getText() with or without transformation to match: is "I HAVE SAVED \n THE SEED" and view has effective visibility <VISIBLE>)


    View Hierarchy:
    +>DecorView{id=-1, visibility=VISIBLE, width=1440, height=2560, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params={(0,0)(fillxfill) sim={adjust=resize} ty=BASE_APPLICATION wanim=0x10302fd
      fl=LAYOUT_IN_SCREEN LAYOUT_INSET_DECOR SPLIT_TOUCH HARDWARE_ACCELERATED DRAWS_SYSTEM_BAR_BACKGROUNDS
      pfl=NO_MOVE_ANIMATION FORCE_DRAW_STATUS_BAR_BACKGROUND FIT_INSETS_CONTROLLED
      bhv=DEFAULT
      fitSides=}, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=3}
    |
    +->LinearLayout{id=-1, visibility=VISIBLE, width=1440, height=2476, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@2a22930, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=2}
    |
    +-->ViewStub{id=16908741, res-name=action_mode_bar_stub, visibility=GONE, width=0, height=0, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=true, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@2b859a9, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0}
    |
    +-->FrameLayout{id=-1, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@65a86cf, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=84.0, child-count=1}
    |
    +--->FitWindowsLinearLayout{id=2131230776, res-name=action_bar_root, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@3187c3a, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=2}
    |
    +---->ViewStubCompat{id=2131230787, res-name=action_mode_bar_stub, visibility=GONE, width=0, height=0, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=true, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@622c0eb, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0}
    |
    +---->ContentFrameLayout{id=16908290, res-name=content, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@455ee48, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +----->ReactRootView{id=21, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@6022206, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +------>ReactViewGroup{id=39, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@bea80c7, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +------->RNGestureHandlerRootView{id=37, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@6ffa6f4, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +-------->SafeAreaProvider{id=35, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@4709e1d, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +--------->ReactViewGroup{id=33, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@b8e5892, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=2}
    |
    +---------->ScreenContainer{id=27, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@2ea3e60, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +----------->ScreensFrameLayout{id=-1, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@8356bde, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +------------>Screen{id=25, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@b6673d5, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=2}
    |
    +------------->ReactViewGroup{id=3, visibility=VISIBLE, width=1440, height=0, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@da867ea, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=0}
    |
    +------------->ReactViewGroup{id=23, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@959badb, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +-------------->ReactViewGroup{id=19, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@2857978, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +--------------->ReactViewGroup{id=17, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@8a77851, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +---------------->ReactViewGroup{id=15, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@59d18b6, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=2}
    |
    +----------------->ReactViewGroup{id=59, visibility=VISIBLE, width=1440, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@42f29b7, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=1}
    |
    +------------------>ReactScrollView{id=125, visibility=VISIBLE, width=1187, height=2392, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@e1a488d, tag=null, root-is-layout-requested=false, has-input-connection=false, x=127.0, y=0.0, child-count=1}
    |
    +------------------->ReactViewGroup{id=123, visibility=VISIBLE, width=1187, height=2041, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@d5a0a42, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=8}
    |
    +-------------------->ReactTextView{id=65, visibility=VISIBLE, width=397, height=186, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@9b9ff90, tag=null, root-is-layout-requested=false, has-input-connection=false, x=395.0, y=70.0, text=Zingo!, input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactTextView{id=69, visibility=VISIBLE, width=283, height=71, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@96fe089, tag=null, root-is-layout-requested=false, has-input-connection=false, x=452.0, y=256.0, text=v0.1.3 (119), input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactImageView{id=73, visibility=VISIBLE, width=350, height=350, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@e02888e, tag=null, root-is-layout-requested=false, has-input-connection=false, x=419.0, y=362.0}
    |
    +-------------------->ReactTextView{id=79, visibility=VISIBLE, width=333, height=71, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@88398af, tag=null, root-is-layout-requested=false, has-input-connection=false, x=427.0, y=887.0, text=Actual server:, input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactTextView{id=85, visibility=VISIBLE, width=914, height=71, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@ee784bc, tag=null, root-is-layout-requested=false, has-input-connection=false, x=137.0, y=969.0, text=https://mainnet.lightwalletd.com:9067, input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactViewGroup{id=95, visibility=VISIBLE, width=600, height=182, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@86cfc45, tag=null, root-is-layout-requested=false, has-input-connection=false, x=294.0, y=1075.0, child-count=1}
    |
    +--------------------->ReactTextView{id=89, visibility=VISIBLE, width=446, height=76, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@da29f9a, tag=null, root-is-layout-requested=false, has-input-connection=false, x=77.0, y=53.0, text=CHANGE SERVER, input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactViewGroup{id=105, visibility=VISIBLE, width=893, height=182, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@5fb10cb, tag=null, root-is-layout-requested=false, has-input-connection=false, x=147.0, y=1327.0, child-count=1}
    |
    +--------------------->ReactTextView{id=99, visibility=VISIBLE, width=739, height=142, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@30630a8, tag=null, root-is-layout-requested=false, has-input-connection=false, x=77.0, y=20.0, text=CREATE NEW WALLET (NEW SEED), input-type=0, ime-target=false, has-links=false}
    |
    +-------------------->ReactViewGroup{id=115, visibility=VISIBLE, width=942, height=182, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@66617c1, tag=null, root-is-layout-requested=false, has-input-connection=false, x=123.0, y=1754.0, child-count=1}
    |
    +--------------------->ReactTextView{id=109, visibility=VISIBLE, width=788, height=76, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@4fb1b66, tag=null, root-is-layout-requested=false, has-input-connection=false, x=77.0, y=53.0, text=RESTORE WALLET FROM SEED, input-type=0, ime-target=false, has-links=false}
    |
    +----------------->ReactViewGroup{id=9, visibility=VISIBLE, width=1440, height=0, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@827eea7, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=0}
    |
    +---------->ReactViewGroup{id=29, visibility=VISIBLE, width=1440, height=280, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@fe1af54, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0, child-count=0}
    |
    +->View{id=16908336, res-name=navigationBarBackground, visibility=VISIBLE, width=1440, height=84, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@57b6efd, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=2476.0}
    |
    +->View{id=16908335, res-name=statusBarBackground, visibility=VISIBLE, width=1440, height=84, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.FrameLayout$LayoutParams@60d87f2, tag=null, root-is-layout-requested=false, has-input-connection=false, x=0.0, y=0.0}

      18 |     let newWalletButton = element(by.text("CREATE NEW WALLET (NEW SEED)"));
      19 |     await newWalletButton.tap();
    > 20 |     await element(by.text("I HAVE SAVED \n THE SEED")).tap();
         |                                                        ^
      21 |     // For some reason, the element that says 'ZEC', according to react,
      22 |     // has text of ' ---'
      23 |     // let zecText = element(by.text(" ---")).atIndex(1);

      at createErrorWithUserStack (node_modules/detox/src/utils/errorUtils.js:36:27)
      at Client._doSendAction (node_modules/detox/src/client/Client.js:164:32)
      at Client._sendMonitoredAction (node_modules/detox/src/client/Client.js:157:25)
      at Client.sendAction (node_modules/detox/src/client/Client.js:143:14)
      at Client.execute (node_modules/detox/src/client/Client.js:132:25)
      at InvocationManager.execute (node_modules/detox/src/invoke.js:12:39)
      at ActionInteraction.execute (node_modules/detox/src/android/interactions/native.js:23:31)
      at NativeElement.tap (node_modules/detox/src/android/core/NativeElement.js:37:97)
      at Object.tap (e2e/my.test.js:20:56)
      at asyncGeneratorStep (node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
      at _next (node_modules/@babel/runtime/helpers/asyncToGenerator.js:22:9)

14:05:29.229 detox[343380] E lifecycle Command failed with exit code = 1:
jest --config e2e/jest.config.js
14:05:29.231 detox[343380] i ws-server Detox server has been closed gracefully
14:05:29.232 detox[343380] E ws-server connection :56180<->:56283
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

C:\Users\juank\github\zingo-mobile-background-sync-android>

Reply:
Great! It looks like on your device, the test isn't properly continuing to the seed-displaying screen (after the tests clicks "CREATE NEW WALLET (NEW SEED)", but it's staying on that screen.. Not sure why it's doing that, but it's enough of a place to start I think I can start debugging this!